feature/floormap/src/main/java/io/github/droidkaigi/confsched2020/floormap/ui/FloorMapFragment.kt

```diff
@@ -29,3 +46,18 @@ class FloorMapFragment : Fragment(R.layout.fragment_floormap), Injectable {
         }
     }
 }
+
+@DrawableRes
+fun Room.getRoomTypeResource(): Int {
+    val type = this.roomType ?: return R.drawable.ic_floor_map
+    return when (type) {
+        Room.RoomType.EXHIBITION -> R.drawable.ic_floor_map_exhibition
```

Comment:
It is a dummy image, but there is a possibility that it will be released without being changed, so can you please delete the xml and put out a normal map image this time?
// TODO: Add pin images

Reply:
@takahirom 
It is only EXHIBITION? or everything 👀?

Reply:
I fixed with 3340d74